### ISSUE TITLE
GL: 4 Elements HD invalid dimension 

### ISSUE THREAD

**Issue by raven02** [RPCS3/rpcs3#2096]
Vulkan and DX12 able to in-game. Interesting only GL backend hit the sizing 0 error .
 

```
F {rsx::thread} class std::runtime_error thrown: Invalid arguments (addr=0xc2e70000, size=0x0)
(in file Emu\Memory\vm.cpp:375)
```

**Comment by raven02** [comment]
It hits the vm::page_protect()

		if (!size || (size | addr) % 4096)
		{
			fmt::throw_exception("Invalid arguments (addr=0x%x, size=0x%x)" HERE, addr, size);
		}

**Comment by raven02** [comment]
Fixed by #2097 and 4 Elements HD is fully working using GL backend.
